lib/git.js

```diff
@@ -1,92 +1,108 @@
-const shell = require('shelljs');
-const fs = require('fs');
-const path = require('path');
-const chalk = require('chalk');
+const shell = require("shelljs");
+const fs = require("fs");
+const path = require("path");
+const chalk = require("chalk");
 
 // utils
 const generateRemoteLinkSsh = (username, projectName) => {
-  return `git@github.com:${username}/${projectName}.git`
-}
+  return `git@github.com:${username}/${projectName}.git`;
+};
 const generateRemoteLinkHttps = (username, projectName) => {
-  return `https://github.com/${username}/${projectName}.git`
-}
+  return `https://github.com/${username}/${projectName}.git`;
+};
 const generateRemoteLink = (username, projectName, connectionType) => {
-  if (connectionType === 'https') {
+  if (connectionType === "https") {
     return generateRemoteLinkHttps(username, projectName);
   } else {
     return generateRemoteLinkSsh(username, projectName);
   }
-}
+};
 
 // Native actions
 const init = () => {
-  shell.exec('git init');
-}
+  shell.exec("git init");
```

Comment:
is there any merit to doing this ourselves?
why not use https://www.npmjs.com/package/simple-git or something like it?